server_config.cc

```diff
@@ -4,54 +4,92 @@
 #include <string>
 
 ServerConfig::ServerConfig(NginxConfig config) {
-  fillOutMap(config);
+  std::vector<std::string> basePath = {};
+  fillOutMap(config, basePath);
+  printPropertiesMap();
 }
 
-ServerConfig::~ServerConfig() {
-  // TODO
+ServerConfig::~ServerConfig() {}
+
+// TODO: is this a terrible function name?
+// Helper function to concatenate multi-word config parameter names
+std::string buildWordyParam(std::shared_ptr<NginxConfigStatement> statement,
+                            size_t valueSize) {
```

Comment:
Explained in [this commit](https://github.com/UCLA-CS130/Mr.-Robot-et-al./pull/37/commits/5cb8eb26f24ac510f72d57d987b4e2f6d7438f06)!